.github/workflows/ci.yml

```diff
@@ -80,7 +80,7 @@ jobs:
                 run: cargo install wasm-pack
 
             -   name: Run tests default features
-                run: wasm-pack test --node --features rust_crypto
+                run: wasm-pack test --node --features rust_crypto,getrandom/js
```

Comment:
There's a related issue: https://github.com/Keats/jsonwebtoken/issues/431
But I don't know enough about wasm to know what to actually do

Reply:
I think this PR might actually help with that, given `getrandom/js` is now only enabled in CI. The user may be able to enable another wasm-compatible `getrandom` backend after this lands.